db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
```

Comment:
Also you can sneak around AR validations in the console. The DB is like the
Honey Badger in it's level of regard for others.



-- 
@adarshp <http://www.twitter.com/adarshp>
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal

On Tue, Nov 10, 2015 at 7:48 PM, Gabriel Berke-Williams <
notifications@github.com> wrote:

> In db/migrate/20151110191852_create_redirections.rb
> <https://github.com/gabebw/hotline-webring/pull/1#discussion_r44499270>:
>
> > @@ -0,0 +1,10 @@
> > +class CreateRedirections < ActiveRecord::Migration
> > +  def change
> > +    create_table :redirections do |t|
> > +      t.timestamps null: false
> > +      t.references :next
>
> I like DB validations for uniqueness because they actually provide a
> service: AR uniqueness validations can have race conditions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gabebw/hotline-webring/pull/1/files#r44499270>.
>